Source/Classes/TimeZones.cs

```diff
@@ -0,0 +1,165 @@
+using System.Diagnostics.CodeAnalysis;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace vsteam_lib
+{
+   public static class TimeZones
+   {
+
+      public static Dictionary<string, string> GetTimeZones()
+      {
+
+         var timeZones = new Dictionary<string, string>(){
```

Comment:
I took these values straight from Azure DevOps UI. They are used everywhere where you have time zones (e.g. pool maintenance, user profile time& region etc.). So this is generalized.

![image](https://user-images.githubusercontent.com/7256168/111078495-02479900-84f6-11eb-8753-6299b7d4388a.png)